packages/quick_actions/quick_actions_ios/example/pubspec.yaml

```diff
@@ -25,3 +25,8 @@ dev_dependencies:
 
```

Comment:
Can you also update the example app to use `localizedSubtitle` like you did in `packages/quick_actions/quick_actions/example/lib/main.dart`?

Around here:
https://github.com/flutter/packages/blob/67d8b50c1796eefa948a1367c1184b0e71753e44/packages/quick_actions/quick_actions_ios/example/lib/main.dart#L53-L54

```dart
        localizedTitle: 'Action one',
        localizedSubtitle: 'Action one subtitle',
        icon: 'AppIcon',
```

Then the native test here
https://github.com/flutter/packages/blob/67d8b50c1796eefa948a1367c1184b0e71753e44/packages/quick_actions/quick_actions_ios/example/ios/RunnerUITests/RunnerUITests.swift#L77-L78

becomes:
```swift
    findAndTapQuickActionButton(
      buttonName: "Action one, Action one subtitle", quickActionsAppIcon: quickActionsAppIcon, springboard: springboard)

```
In the XCUITest you can `p springboard` to see the label is joining the title and subtitle with a comma:
```
                      Button, 0x107906630, {{31.0, 260.7}, {250.0, 63.7}}, identifier: 'action_one', label: 'Action one, Action one subtitle'
```